pkg/kv/kvserver/helpers_test.go

```diff
@@ -91,10 +90,18 @@ func (s *Store) ComputeMVCCStats() (enginepb.MVCCStats, error) {
 
 // ConsistencyQueueShouldQueue invokes the shouldQueue method on the
 // store's consistency queue.
-func (s *Store) ConsistencyQueueShouldQueue(
-	ctx context.Context, now hlc.Timestamp, r *Replica, cfg *config.SystemConfig,
+func ConsistencyQueueShouldQueue(
```

Comment:
why is this exported?

Reply:
(and why does it even exist? On master, it's called only from `TestConsistencyQueueRequiresLive` which you've refactored).

Reply:
TestConsistencyQueueRequiresLive stills calls this and it lives in kvserver_test. Unfortunately we cant move that file into kvserver because of the circular dependency between TestCluster and kvserver.

Reply:
Makes sense!